src/compositor/seat/seat.c

```diff
@@ -114,8 +114,10 @@ seat_handle_key(struct wlc_seat *seat, const struct wlc_input_event *ev)
       return;
 
    wlc_keyboard_update_modifiers(&seat->keyboard, ev->device);
+   struct wlc_modifiers mods = {0, 0};
```

Comment:
Yes. If I use keyboard->modifiers, we get the wrong keysym, because it includes the ctrl+alt too.